R/formatRt.R

```diff
@@ -0,0 +1,36 @@
+##' Converts seconds to/from "min:sec" format.
+##'
+##' @title Format Retention Time
+##'
+##' @param rt The retention time, either a `numeric()` in seconds or a
+##'     `character()` as `"mm:ss"`.
+##' 
+##' @return A reformatted retention time.
+##' 
+##' @author Laurent Gatto
+##'
+##' @export
+##'
+##' @examples
+##' formatRt(1524)
+##' formatRt(1)
+##' formatRt("25:24")
```

Comment:
I was wondering, does this also support `rt` to be of length > 1?

Reply:
It does:
```r
> formatRt(1:10)
 [1] "0:01" "0:02" "0:03" "0:04" "0:05" "0:06" "0:07" "0:08" "0:09" "0:10"
```
I'll mention it in the man page and in the example.

Reply:
Addressed in latest addition to this PR.

In addition, I also now require an input of length > 0.